flocker/node/agents/gce.py

```diff
@@ -417,18 +436,18 @@ def _do_blocking_operation(self, function, timeout_sec=60, **kwargs):
             resource dict as described above.
         :param int timeout_sec: The maximum amount of time to wait in seconds
             for the operation to complete.
+        :param sleep: A callable that has the same signature and function as
+            ``time.sleep``. Only intended to be used in tests.
         :param kwargs: Additional keyword arguments to pass to function.
 
         :returns dict: A dict representing the concluded GCE operation
             resource.
         """
-        timeout = kwargs.pop('timeout', VOLUME_DEFAULT_TIMEOUT)
-        sleep = kwargs.pop('sleep', None)
         args = dict(project=self._project, zone=self._zone)
         args.update(kwargs)
         operation = function(**args).execute()
         return wait_for_operation(
-            self._compute, operation, [1]*timeout, sleep)
+            self._compute, operation, [1]*timeout_sec, sleep)
```

Comment:
@justnoise would you mind glancing at this? I think timeout_sec was unused after something, and so when I merged master in I think I did a drive-by fix of that. Care to double check I did it right?